app/api/intendedGraduations/delete/func.js

```diff
@@ -0,0 +1,20 @@
+const app = require('@/app')
+const errors = require('@/errors')
+
+module.exports = async function(context){
+  let { graduationId } = context.params;
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
  let { graduationId } = context.params
```